rusk-wallet/src/bin/interactive.rs

```diff
@@ -487,32 +298,142 @@ fn menu_op(
     let cmd = cmd_menu.answer(&answer).to_owned();
 
     let res = match cmd {
-        CMI::PhoenixTransactions => {
-            transaction_op_menu_phoenix(profile_idx, phoenix_balance)?
+        CMI::Transfer => {
+            let rcvr = prompt::request_rcvr_addr("recipient")?;
+            let (sender, balance) = match &rcvr {
+                Address::Shielded(_) => {
+                    (wallet.shielded_address(profile_idx)?, phoenix_balance)
+                }
+                Address::Public(_) => {
+                    (wallet.public_address(profile_idx)?, moonlight_balance)
+                }
+            };
```

Comment:
Don't you need to check here if the `rcvr` and `sender` addresses match the transaction protocol?

Reply:
alternatively you can first see if the sender is public or shielded and according to that request a public or shielded receiver, so split `request_rcvr_addr` into public and shielded.

Reply:
> Don't you need to check here if the rcvr and sender addresses match the transaction protocol?

No, we're in interactive mode here, and the sender is chosen according to the receiver using the same profile

> alternatively you can first see if the sender is public
We never ask for the sender in interactive